packages/@o3r/core/schematics/service/index.ts

```diff
@@ -45,11 +44,16 @@ function ngGenerateServiceFn(options: NgGenerateServiceSchematicsSchema): Rule {
     const workspaceConfig = getWorkspaceConfig(tree);
     let packageName = destination;
     if (workspaceProject?.projectType !== 'application') {
-      let rec = '..';
-      while (!tree.exists(path.resolve(destination, rec, 'package.json')) && path.resolve(destination, rec) !== '/') {
-        rec = path.join('..', rec);
+      let currentDirectory = path.join(destination);
+      let parent = '..';
```

Comment:
the value never change, it should be `const`.
(I am surprised it has not been caught by the linter :S)

Reply:
Fixed